acq4/devices/PMT/PMT.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from acq4.devices.Device import *
+from acq4.devices.OptomechDevice import OptomechDevice
+from acq4.devices.DAQGeneric import DAQGeneric
+
+class PMT(DAQGeneric, OptomechDevice):
```

Comment:
OK. 

And which relation of the current `PMT` device to `PhotoDetector` do you propose? I think this is the part which is not clear to me.